lib/block_view.go

```diff
@@ -520,6 +532,29 @@ func (bav *UtxoView) CopyUtxoView() (*UtxoView, error) {
 		newView.TransactorNonceMapKeyToTransactorNonceEntry[entryKey] = &newEntry
 	}
 
+	// Copy the LockedBalanceEntries
+	newView.LockedBalanceEntryKeyToLockedBalanceEntry = make(map[LockedBalanceEntryKey]*LockedBalanceEntry,
+		len(bav.LockedBalanceEntryKeyToLockedBalanceEntry))
+	for entryKey, entry := range bav.LockedBalanceEntryKeyToLockedBalanceEntry {
+		newEntry := *entry
+		newView.LockedBalanceEntryKeyToLockedBalanceEntry[entryKey] = &newEntry
+	}
+
+	// Copy the LockupYieldCurvePoints
+	newView.PKIDToLockupYieldCurvePointKeyToLockupYieldCurvePoints =
+		make(map[PKID]map[LockupYieldCurvePointKey]*LockupYieldCurvePoint, len(bav.PKIDToLockupYieldCurvePointKeyToLockupYieldCurvePoints))
```

Comment:
Shorten lines like these. Your editor should have a line in it that tells you what the cutoff is. When lines are long like this it makes reviewing the code annoying for others.